requirements.txt

```diff
@@ -1,2 +1,3 @@
-pennylane>=0.6
-cirq>=0.6
+pennylane>=0.7
+cirq>=0.7
+numpy~=1.16
```

Comment:
This is required as else there might be errors arising from the numpy version used
https://github.com/quantumlib/Cirq/blob/8612933e2dc8f4db1392187a72a2ecd132f6cee8/requirements.txt#L7
```python
            if dist not in req:
                # Oops, the "best" so far conflicts with a dependency
                dependent_req = required_by[req]
>               raise VersionConflict(dist, req).with_context(dependent_req)
E               pkg_resources.ContextualVersionConflict: (numpy 1.15.4 (/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages), Requirement.parse('numpy~=1.16'), {'cirq'})
../../../virtualenv/python3.7.1/lib/python3.7/site-packages/pkg_resources/__init__.py:791: ContextualVersionConflict
 TestStatePreparationErrorsNonAnalytic.test_qubit_state_vector_not_analytic_error[100-False] 
```